recipes/eups/build.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+EUPS_HOME="${PREFIX}/eups"
+EUPS_DIR="${EUPS_HOME}"
+export EUPS_PATH="${PREFIX}/share/eups"
+
+EUPS_PYTHON=$PYTHON  # use PYTHON in the host env for eups
+
+mkdir -p ${EUPS_HOME}
+
+# Install EUPS
+echo -e "\nInstalling EUPS..."
+echo "Using python at ${EUPS_PYTHON} to install EUPS"
+# echo "Configured EUPS_PKGROOT: ${EUPS_PKGROOT}"
+
+mkdir -p "${EUPS_PATH}"/{site,ups_db}
+touch "${EUPS_PATH}/ups_db/.conda_keep"
```

Comment:
You might need to touch the site dir here too. I forget.

Reply:
There should be no need to touch the site dir since it is populated during the build.